vnext/Shared/Modules/BlobModule.cpp

```diff
@@ -0,0 +1,454 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "BlobModule.h"
+
+#include <Modules/CxxModuleUtilities.h>
+#include <Modules/IHttpModuleProxy.h>
+#include <Modules/IWebSocketModuleProxy.h>
+#include <ReactPropertyBag.h>
+#include <unicode.h>
+
+// React Native
+#include <cxxreact/JsArgumentHelpers.h>
+
+// Windows API
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Security.Cryptography.h>
+
+// Standard Library
+#include <chrono>
+#include <filesystem>
+#include <fstream>
+
+using namespace facebook::xplat;
+
+using folly::dynamic;
+using std::scoped_lock;
+using std::shared_ptr;
+using std::string;
+using std::vector;
+using std::weak_ptr;
+using winrt::Microsoft::ReactNative::IReactPropertyBag;
+using winrt::Microsoft::ReactNative::ReactNonAbiValue;
+using winrt::Microsoft::ReactNative::ReactPropertyBag;
+using winrt::Microsoft::ReactNative::ReactPropertyId;
+using winrt::Windows::Foundation::GuidHelper;
+using winrt::Windows::Foundation::IInspectable;
+using winrt::Windows::Foundation::Uri;
+using winrt::Windows::Security::Cryptography::CryptographicBuffer;
+
+namespace fs = std::filesystem;
+
+namespace {
+constexpr char moduleName[] = "BlobModule";
+constexpr char blobKey[] = "blob";
+constexpr char blobIdKey[] = "blobId";
+constexpr char offsetKey[] = "offset";
+constexpr char sizeKey[] = "size";
+constexpr char typeKey[] = "type";
+constexpr char dataKey[] = "data";
+
+// See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+const static std::unordered_map<string, const char *> extensionToMime = {
```

Comment:
>extensionToMime = { [](http://example.com/codeflow?start=54&length=19)

This is unused ?

Reply:
Not now, but it will be needed when we implement custom URIs.
Removing for now.